design-reviews/recipes/feature-story-design-review-template.md

```diff
@@ -0,0 +1,60 @@
+# Your Feature or Story Design Title Here
+
+> Does the feature re-use or extend existing patterns / interfaces that have already been established for the project?
+> Does the feature expose new patterns or interfaces that will establish a new standard for new future development?
+
+* [Feature/Story Name](http://link-to-feature-or-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview/Problem Statement
+
+* Describe the feature/story with a high-level summary.
+* Consider additional background and justification, for posterity and historical context.
+
+## Goals/In-Scope
+
+* List the goals that the feature/story will help us achieve.  
+* This should include acceptance criteria required to meet [definition of done](../../team-agreements/definition-of-done/).
+
+## Non-goals / Out-of-Scope
+
+* List the non-goals for the feature/story.
+* This contains work that is beyond the scope of what the feature/component/service is intended for.
+
+## Proposed Design
+
+* Briefly describe the high-level architecture for the feature/story.
+* Relevant diagrams (e.g. sequence, component, context, deployment) should be included here.
+
+## Technology Choices
```

Comment:
Maybe title could be just "Technology" similar to what we did in the milestone design template based on PR feedback?